src/list/manage/components/item-list-panel.css

```
@@ -36,7 +36,7 @@
 .item-filter {
```

So far I've only tried it on MacOS; the following seems to properly align the label:

```suggestion
.second-row select {
  line-height: 34px;
}

.item-filter {
```